environment.yml

```diff
@@ -98,7 +98,7 @@ dependencies:
   - pillow=6.0.0=py37h34e0f95_0
   - pip=19.2.2=py37_0
   - pluggy=0.12.0=py_0
-  - pre_commit=1.17.0=py37_0
+  - pre_commit=1.11.2=0
```

Comment:
I have worked out what the pre-commit problem was. The update from pre-commit 1.11.2 -> 1.12 broke the reorder_python_imports integration. 
So I've conda installed the most up-to-date version possible (but this updates a few other core packages hence the edits here)